containers/podman-desktop
# chore: update airgap installer for windows to pull images from quay.io

### What does this PR do?
Updates airgap installer to fetch podman from quay.io
Related info: https://github.com/containers/podman/pull/23949#issuecomment-2355128723

- [x] rebase after https://github.com/podman-desktop/podman-desktop/pull/9882 will be merged
### What issues does this PR fix or reference?
Closes #8917 

### How to test this PR?


Run command `AIRGAP_DOWNLOAD=1 pnpm compile:current` and you should get 2 .zst files in extensions\podman\packages\extension\assets folder, run the PD using `pnpm watch` command and in the end try to create podman machine with no connection to internet (it should use those 2 files to create the wsl machine)
